lib/core/src/Cardano/Wallet/Primitive/Types.hs

```diff
@@ -1324,6 +1327,24 @@ newtype ChimericAccount = ChimericAccount { unChimericAccount :: ByteString }
 
 instance NFData ChimericAccount
 
+-- | Represent a delegation certificate.
+data DelegationCertificate
+    = CertDelegateNone ChimericAccount
+    | CertDelegateFull ChimericAccount PoolId
+    deriving (Generic, Show, Eq, Ord)
```

Comment:
Hi @KtorZ: Does the protocol allow for a certificate that cancels all stake delegation for an account?

I was under the impression that _delegating to a non-existing pool_ was the only way to achieve this.

If that is still true, then is it necessary to have the `CertDelegateNone` constructor?

Reply:
Answered in a previous comment :)